e2e/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "tutorialkit-e2e",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
```

Comment:
Nit: we can probably remove those two scripts?

Reply:
We should leave at least one of these. When developing test cases it's really nice to get the server running when making changes to content and templates.

I think we could remove `start` here. I added it as I was running both dev/start from time to time - mixed conventions across projects.